docs/source/setup.rst

```diff
@@ -13,16 +13,21 @@ E.g. ``Post`` model, as shown below:
 .. code:: python
 
     from django.contrib.contenttypes.fields import GenericRelation
+    from django.urls import reverse
 
     from comment.models import Comment
 
     class Post(models.Model):
         author = models.ForeignKey(User)
         title = models.CharField(max_length=200)
+        slug = models.SlugField(unique=True)
         body = models.TextField()
         # the field name should be comments
         comments = GenericRelation(Comment)
 
+        def get_absolute_url(self):
+            return reverse('post_detail_url', kwargs={'slug': self.slug})
+
```

Comment:
Maybe you can add a note that emphasizes that adding this method is important. Something of the form below seems more useful to me.

```rst
.. important:: 

    Please add a ``get_absolute_url`` method to the model that you're associating with the ``Comment`` model. It is used at several places.

    .. code:: python

        from django.urls import reverse
        
        def get_absolute_url(self):
            return reverse('post_detail_url', kwargs={'slug': self.slug})

Reply:
Thanks for the feedback